collaboration/README.md

```diff
@@ -1,102 +1,61 @@
 # Group Norms
 
 These are rules we abide by as a team. It is a living document and hence can be updated
-to reflect current situation. It's important that team members revist it frequently.
+to reflect the current situation. It's important that team members revisit it frequently.
 
-<details>
-<summary> How we build and maintain cognitive and affective trust? </summary>
+## How we build and maintain cognitive and affective trust?
 
 - Pay attention when the next person is speaking - conveys a feeling of being respected
-and heard
+  and heard
+- Address the project, not the person - creates a safe environment for achieving
+  common project goal
 
-- Address the project not the person - creates a safe environment for achieving the
-common project goal
-
-</details>
-
-<details>
-<summary> What we do if that trust is broken  </summary>
```

Comment:
The formatting was updated to replace `<details>` and `<summary>` HTML tags with Markdown headings and lists. This ensures compliance with Markdown linting rules and improves compatibility across different Markdown parsers.